tasks/redhat.yml

```diff
@@ -1,12 +1,12 @@
 ---
 - name: 'Installing YUM dependencies'
   yum:
-    pkg: "{{ item }}"
+    pkg:
+      - zip
+      - unzip
+      - perl-DateTime
+      - perl-Sys-Syslog
+      - perl-LWP-Protocol-https
+      - perl-Digest-SHA
     state: present
   with_items:
```

Comment:
Hi @hirokiky , thanks for your contribution. :)

I'll be happy to accept your changes if you remove this `with_items:` line as well.

Reply:
Oh, It's embarrassing... Thanks for your great review!

Reply:
I added below commit to fix it.